db/migrate/20190209235051_create_join_table_order_items.rb

```diff
@@ -0,0 +1,12 @@
+class CreateJoinTableOrderItems < ActiveRecord::Migration[5.1]
+  def change
+    create_table :items do |t|
+      t.references :item_id, foreign_key: true
+      t.references :order_id, foreign_key: true
+      t.integer :order_price
+      t.integer :order_quantity
```

Comment:
I don't know if this is too fancy or not, but could we set up something here such that it enforces a validation - for example, t.order_quantity cannot_be_negative

Nevermind - I just Googled it and it looks complicated.